src/unix/fs.c

```diff
@@ -220,6 +220,93 @@ static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
 }
 
 
+static int uv__fs_mkstemp(uv_fs_t* req) {
+  int r;
+#ifdef UV__O_CLOEXEC
```

Comment:
This needs a rebase, the logic to detect O_CLOEXEC got simplified in https://github.com/libuv/libuv/pull/2454